python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -822,6 +822,52 @@ def test_write_batch_dedup(library_factory):
             assert data_key_version[s] == 0
 
 
+def test_read_batch_time_stamp(arctic_library):
```

Comment:
We should stop adding more tests to `test_arctic.py` it's getting completely unwieldy

Reply:
I agree with you. Yet I think we must test this behavior.

How should we proceed?

Reply:
done in https://github.com/man-group/ArcticDB/pull/651/files#diff-cbe6d7a5a240c5c6068d4319fe88bbfbee9434ff8e4cbc84f6f68b26535005fa